packages/react-components/src/components/Accordion/components/AccordionAnimatedLabel.module.scss

```diff
@@ -14,7 +15,19 @@ $base-class: 'accordion-animated-label';
     max-width: 100%;
 
     &--visible {
-      opacity: 1;
+      animation: change-visibility var(--transition-duration-fast-2) ease-in-out;
+      animation-fill-mode: forwards;
     }
   }
 }
+
+@keyframes change-visibility {
+  0% {
+    opacity: 0;
+  }
+
+  70%,
+  100% {
```

Comment:
why it's done like that? Shouldn't we use shorter duration to make effect faster?

Reply:
I guess the results will be similar in both cases :P